.jenkins/cscs-perftests/ctest.cmake

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) 2020 ETH Zurich
+# Copyright (c) 2017 John Biddiscombe
+#
+# SPDX-License-Identifier: BSL-1.0
+# Distributed under the Boost Software License, Version 1.0. (See accompanying
+# file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+# This is a dummy file to trigger the upload of the perftests reports
+cmake_minimum_required(VERSION 3.1 FATAL_ERROR)
+
+set(CTEST_TEST_TIMEOUT 300)
+set(CTEST_BUILD_PARALLELISM 20)
+set(CTEST_TEST_PARALLELISM 4)
```

Comment:
This one is in principle not needed anymore, since we wouldn't run tests through ctest, and they should run serially anyway.